sdk/core/core-tracing/src/interfaces.ts

```diff
@@ -1,6 +1,36 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * The kind of Span.
+ */
+export enum SpanKind {
```

Comment:
Is this just a duplication of the opentelemetry SpanKind? If so can we write a test to detect if ours drifts from opentelemetry's during dependency upgrades?

Reply:
Can do. 

Also, I did this after you and I walked through the issue "depending on an enum" caused when I tried to do the otel 0.16 update. Perhaps I shouldn't do that universally.

Reply:
And actually, I just realized, I shouldn't need to do this beause all of the packages are referencing opentelemetry/api. I'm going to remove this.

Reply:
Gone!